scripts/extensions/sams/src/components/assets/assetEditorPanel.tsx

```diff
@@ -149,12 +194,27 @@ export class AssetEditorPanelComponent extends React.PureComponent<IProps, IStat
                             key={this.props.original?._id}
                             asset={this.props.original!}
                             onChange={this.onChange}
+                            updates={this.state.updates}
                             fields={[
                                 'name',
                                 'description',
                                 'state',
+                                'tags',
                             ]}
                         />
+                        <FormGroup>
+                            <FormRow>
+                                {this.state.updates?.tags?.map((tag) => (
+                                    <Tag
+                                        key={this.state.updates?.tags?.indexOf(tag)}
+                                        text={tag.name}
+                                        onClick={() => {
+                                            this.removeTag(this.state.updates?.tags?.indexOf(tag)!);
```

Comment:
Again, this would be better off as a function in this class/component, and in turn that function will call `this.props.onChange` with the new tag value

Reply:
In onChange,we pass the tag, but can not pass the operation to be performed. I'll have to pass a string with it that will specify that the tag is needed to be added or removed. I'll be adding the code in commit though.